python/lsst/obs/comCam/ingest.py

```diff
@@ -34,23 +50,57 @@ def getInfo(self, filename):
         if runId != phuInfo['run']:
             raise RuntimeError("Expected runId %s, found %s from path %s" % phuInfo['run'], runId, pathname)
 
-        phuInfo['raftId'] = raftId
-        phuInfo['field'] = acquisitionType
-        phuInfo['jobId'] = int(jobId)
-        phuInfo['raft'] = 'R00'
-        phuInfo['ccd'] = sensorLocationInRaft
+        phuInfo['raftId'] = raftId # also in the header - RAFTNAME
+        phuInfo['field'] = acquisitionType # NOT in the header
+        phuInfo['jobId'] = int(jobId) #  NOT in the header, but do we need it?
```

Comment:
I thought it might be useful to flag these so that I could check back in future to see if they were being used to de-cruft the registry, but perhaps you're right, and I should just de-cruft the code now. Removed 👍